XSUP-41017/AD (#36074)
demisto/content: Packs/Active_Directory_Query/ReleaseNotes/1_6_37.md

#### Integrations

##### Active Directory Query v2

> Fixed an issue where the ***ad-get-user*** command does not pull the accurate PASSWORD_EXPIRED flag.

Reviewer: ```suggestion
Fixed an issue where the ***ad-get-user*** command failed to retrieve the correct PASSWORD_EXPIRED flag.
```
